spec/controllers/ctc/questions/filing_status_controller_spec.rb

```diff
@@ -1,48 +1,35 @@
 require 'rails_helper'
 
-describe Ctc::Questions::FilingStatusController do
-  let(:client) { create :client, tax_returns: [(create :tax_return, filing_status: nil)] }
-  let!(:intake) { create :ctc_intake, client: client }
-
-  before do
-    sign_in intake.client
+describe Ctc::Questions::FilingStatusController, requires_default_vita_partners: true do
+  describe "first page of ctc intake update behavior" do
+    include_context :first_page_of_ctc_intake_update_context, form_name: :ctc_filing_status_form, additional_params: { filing_status: "single" }
+    it_behaves_like :first_page_of_ctc_intake_update
   end
 
   describe '#update' do
     context "with no answer" do
-      let(:params) do
-        {}
-      end
+      include_context :first_page_of_ctc_intake_update_context, form_name: :ctc_filing_status_form, additional_params: { filing_status: nil }
```

Comment:
it seems like, if you really want to only include the context once, you could (instead of passing in additional params) just modify `params` in a before block like so:

```
before do
  params[:ctc_filing_status_form][:filing_status] = nil
end
```

but, this doesn't seem any worse tbh